src/app/components_ngrx/planner-list/planner-list.component.ts

```diff
@@ -183,7 +183,19 @@ export class PlannerListComponent implements OnInit, OnDestroy, AfterViewChecked
 
         if (!queryParams.hasOwnProperty('showCompleted') && !queryParams.showCompleted) {
           this.showCompleted = false;
-          exp['$AND'].push({"state":{"$NE":"closed"}});
+          // not closed state
+          // TODO remove hard coded states and
+          // use meta-states when available
+          const notClosed = this.filterService.queryBuilder(
+            'state',
+            this.filterService.not_in_notation,
+            ['closed', 'Done', 'Removed']
```

Comment:
Even though _Closed_ is not a valid state for any WIT in SCRUM, having it here has no harm. LGTM